assets/js/package-main.js

```diff
@@ -0,0 +1,9 @@
+//= require vendor/jquery.min
+//= require vendor/bootstrap.min
+//= require vendor/cookieconsent.min
+//= require vendor/lazysizes.min
+
+//= require app/main
+//= require app/scroll-to-anchors
+//= require app/tables
+//= require app/custom
```

Comment:
What is the purpose of this file?

Reply:
This file creates a package-main.js which pages should use by default. I've just removed a few js dependencies that you do not need which was causing a console error when I ran the site locally.